doc/user-guide.md

```diff
@@ -107,7 +107,23 @@ After modifying the `*_types.go` file always run the following command to update
 $ operator-sdk generate k8s
 ```
 
-### OpenAPI validation
+### Updating CRD manifests
+
+Now that `MemcachedSpec` and `MemcachedStatus` have fields and possibly annotations, the CRD corresponding to the API's group and kind must be updated. To do so, run the following command:
+
+```console
+$ operator-sdk generate openapi
+```
+
+**Notes:**
+- - Your CRD *must* specify exactly one [storage version][crd-storage-version]. Use the `+kubebuilder:storageversion` [marker][crd-markers] to indicate the GVK that should be used to store data by the API server. This marker should be in a comment above your `Memcached` type.
+- You may see errors like "API rule violation" when running the above command. For information on these errors, read the below section and see the [API rules][api-rules] documentation.
+
+[crd-storage-version]:https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning/#writing-reading-and-updating-versioned-customresourcedefinition-objects
```

Comment:
Should not the links be in the bottom of the file?

Reply:
I actually kind of like these links being at the end of each section since it puts the link closer to its reference. We should only do this when the link is used only in that section though.